templates/projects/assign_users.html

```diff
@@ -11,12 +11,8 @@
     <fieldset>
         <div class="multi-container">
             <select multiple="multiple" name="select_users" id="selected_users">
-                {% for user in all_users %}
-                    {% if user.id in project_users %}
-                        <option value="{{user.id}}" selected>{{user.fullname}}</option>
-                    {% else %}
-                        <option value="{{user.id}}">{{user.fullname}}</option>
-                    {% endif %}
+                {% for user in all_users|sort(attribute='last_name') if user.enabled %}
```

Comment:
maybe sort by `fullname` in the case of a `last_name` conflict by chaining sorts, like:
```
{% for user in all_users|sort(attribute='last_name')|sort(attribute='fullname') if user.enabled %}
```

Reply:
Great suggestion. Indeed, `last_name` will [default](https://github.com/bloomberg/pybossa/blob/0b62831adce6f01614ed2197de4ef90511c724c0/pybossa/view/projects.py#L3870) to `fullname` when there is no space in the name.